packages/duckdb/test_duckdb.py

```diff
@@ -0,0 +1,19 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["duckdb", "pandas"])
```

Comment:
You need to declare `pandas` as a requirement in the meta.yaml file. Look at the meta.yaml files for other packages to see how.
```suggestion
@run_in_pyodide(packages=["duckdb"])
```

Reply:
Probably we need to add an option to skip the test if `run_in_pyodide` requires packages that are not built.

Reply:
Perhaps, but in this case it's a mistake.

Reply:
Pandas is not a requirement, though, it's just needed for this test? I'd guess duckdb doesn't _require_ pandas, but just supports it?

Reply:
Pandas isn't a requirement looking at what happens when you install duckdb in an empty env. However numpy *is*. I'll update the meta.yaml and update the test to remove pandas as a requirement